sprocs/assessment_instances_select_log.sql

```diff
@@ -342,6 +342,68 @@ BEGIN
                     asl.assessment_instance_id = ai_id
             )
             UNION
+            (
+                SELECT
+                    7.5 AS event_order,
+                    'Time limit expiry'::TEXT AS event_name,
+                    'gray3'::TEXT AS event_color,
+                    asl.date_limit AS date,
+                    u.user_id AS auth_user_id,
+                    u.uid AS auth_user_uid,
```

Comment:
What does it mean for a time limit expiry to have an associated user?

Reply:
The logs require a `auth_user_id` and `auth_user_uid` columns for all queries, in this case the query is set to whoever set the time limit.